generators/plugin/index.js

```diff
@@ -98,7 +105,7 @@ module.exports = yeoman.Base.extend({
 
     this.registerTransformStream(filters);
 
-    this.fs.copyTpl([`${this.templatePath()}/**/*`, adminFiles, publicFiles], this.destinationPath(), _props);
+    this.fs.copyTpl([`${this.templatePath()}/**/*`, adminFiles, publicFiles, composerFile], this.destinationPath(), _props);
```

Comment:
Do we really need this? I haven't run the actual code, but I think `${this.templatePath()}/**/*` would match the `composer.json` file as well.